docs/src/main/sphinx/installation/cli.rst

```diff
@@ -100,7 +100,8 @@ the following table must be entered in uppercase. The default value is ``CSV``.
 Examples
 ^^^^^^^^
 
-Consider the following command run as shown, or with ``--output-format CSV``:
+Consider the following command run as shown, or with the
+``--output-format=CSV`` option, which is the default for non-interactive:
```

Comment:
non-interactive usage of the CLI

or 

non-interactive invocation

or something to finish the sentence